NAPS2.Escl/Client/EsclClient.cs

```diff
@@ -101,10 +101,15 @@ public async Task<EsclJob> CreateScanJob(EsclScanSettings settings)
         response.EnsureSuccessStatusCode();
         Logger.LogDebug("POST OK");
 
-        return new EsclJob
-        {
-            UriPath = response.Headers.Location!.AbsolutePath
-        };
+        return response.Headers.Location!.IsAbsoluteUri
+            ? new EsclJob
+            {
+                UriPath = response.Headers.Location!.AbsolutePath
```

Comment:
Let's put the ternary inside here to simplify, e.g. `UriPath = uri.IsAbsoluteUri ? uri.AbsolutePath : uri.OriginalString`